packages/reactivity/src/computed.ts

```diff
@@ -1,14 +1,14 @@
 import { effect, ReactiveEffect, activeReactiveEffectStack } from './effect'
-import { UnwrapNestedRefs } from './ref'
+import { Ref, UnwrapNestedRefs } from './ref'
 import { isFunction } from '@vue/shared'
 
-export interface ComputedRef<T> {
+export interface ComputedRef<T> extends Ref<T> {
   _isRef: true
```

Comment:
Since it now extends `Ref<T>`, we can remove `_isRef: true` here. Same for `WritableComputedRef` below.